bin/server.js

```diff
@@ -19,9 +19,11 @@ var Mailer = require('../mailer')(legacyMailerLog)
 
 var dbConnect = require('../lib/db_connect')()
 
+var DEFAULT_LANGUAGE = 'en'
+
 P.all(
   [
-    require('../translator')(config.get('locales')),
+    require('../translator')(config.get('locales'), DEFAULT_LANGUAGE),
```

Comment:
Does `DEFAULT_LANGUAGE` have to exists in `config.get('locales')`? Or will `en` always be a locale?